### DISCUSSION TITLE
Any interest in a UI?

### DISCUSSION THREAD

**Discussion by bmarsh9** [procrastinate-org/procrastinate#752] [Ideas]
I've been exploring this library for a side project and I've had to create a small UI to view tasks and jobs. Checking if there is any interest in this? I know there are bigger issues to tackle.. but I'm happy to share the code if its helpful - snippets below

### Table with jobs
![procrastinate_1](https://user-images.githubusercontent.com/26391921/221255574-a28ba24d-6a36-49e1-a19b-b42c87ec20c2.PNG)

### On click of job, show more details and events
![procrastinate_2](https://user-images.githubusercontent.com/26391921/221255591-c47dd350-6d2c-4c35-9f00-618f1ee0c349.PNG)

**Comment by ewjoachim** [comment]
Absolutely, I'm also quite surprised that we never did a ticket on this, but is would be awesome !

**Comment by ducdetronquito** [comment]
[River](https://github.com/riverqueue/river), an equivalent of procrastinate for Go, [just released a management UI](https://riverqueue.com/blog/announcing-river-ui).

I just post it here for inspiration to anyone trying anything UI related for procrastinate :)

**Reply by ewjoachim** [reply]
Awesome :)
